src/kinetics/InterfaceKinetics.cpp

```diff
@@ -499,12 +499,33 @@ void InterfaceKinetics::getDeltaSSGibbs(doublereal* deltaGSS)
     // of the solution.
     for (size_t n = 0; n < nPhases(); n++) {
         thermo(n).getStandardChemPotentials(m_mu0.data() + m_start[n]);
+        //thermo(n).getGibbs_RT_ref(m_grt.data() + m_start[n]);
     }
 
     // Use the stoichiometric manager to find deltaG for each reaction.
+    //getReactionDelta(m_mu0.data(), deltaGSS);
     getReactionDelta(m_mu0.data(), deltaGSS);
 }
 
+void InterfaceKinetics::getDeltaRefGibbs(doublereal * deltaGSS)
+{
+    // Get the standard state chemical potentials of the species. This is the
+    // array of chemical potentials at unit activity We define these here as the
+    // chemical potentials of the pure species at the temperature and reference 
+    // pressure of the solution.
```

Comment:
State the numerical value of the reference pressure as well.

Reply:
Done